src/ApiService/ApiService/QueueTaskHeartbeat.cs

```diff
@@ -20,14 +20,14 @@ public class QueueTaskHearbeat {
 
     [Function("QueueTaskHeartbeat")]
     public async Async.Task Run([QueueTrigger("task-heartbeat", Connection = "AzureWebJobsStorage")] string msg) {
-        _logger.LogInformation($"heartbeat: {msg}");
+        _logger.LogInformation("heartbeat: {Message}", msg);
```

Comment:
These are structured logging messages and the string should be fixed with named parameters, see e.g.: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.logcritical?view=dotnet-plat-ext-6.0#microsoft-extensions-logging-loggerextensions-logcritical(microsoft-extensions-logging-ilogger-system-string-system-object())